curvefs/src/tools/curvefs_tool.h

```diff
@@ -232,9 +232,16 @@ class CurvefsToolRpc : public CurvefsTool {
             }
             // if service_stub_func_ does not assign a value
             // it will crash in there
-            service_stub_func_(controller_.get(), &requestQueue_.front(),
-                               response_.get());
-            if (controller_->Failed()) {
+            uint32_t i = 0;
+            for (i = 0; i <= FLAGS_rpcRetryTimes; ++i) {
+                controller_->Reset();
+                service_stub_func_(controller_.get(), &requestQueue_.front(),
+                                   response_.get());
+                if (!controller_->Failed()) {
```

Comment:
it's better to check the error code of the controller, otherwise, you will do meaningless retry if the server is quitted or never started.

you can see this for some help

https://github.com/opencurve/curve/blob/ba87f7dba7dfccb4b78ab9bca54e1c261783c41a/src/client/mds_client.cpp#L125

Reply:
> it's better to check the error code of the controller, otherwise, you will do meaningless retry if the server is quitted or never started.
> 
> you can see this for some help
> 
> https://github.com/opencurve/curve/blob/ba87f7dba7dfccb4b78ab9bca54e1c261783c41a/src/client/mds_client.cpp#L125

fix